mne/minimum_norm/inverse.py

```diff
@@ -1177,6 +1177,14 @@ def apply_inverse_epochs(epochs, inverse_operator, lambda2, method="dSPM",
         Additional options for eLORETA. See Notes of :func:`apply_inverse`.
 
         .. versionadded:: 0.16
+    delayed : bool
+        If False, the full data is returned. If True, data is stored as a tuple
+        of smaller arrays in order to save memory. In this case, the full data
+        field will be automatically constructed when stc.data is called for the
+        first time. `delayed=True` is only implemented for fixed
+        orientations. Defaults to False.
```

Comment:
does it work for any pick_ori ? please specify that data is the source time courses. What are the tuple of smaller arrays? I feel you can do more pedagogy here.

Reply:
I didn't want to make the param dostring too long.
For explaining the kernel/sens_data stuff, I thought about something like:
See :class:`mne.SourceEstimate`.data but i'm not sure whether and how you can directly link to class attributes in the documentation. Can you maybe help me with this?
Or should i just explain it directly in the docstring?